test/test-handler-spec.js

```diff
@@ -68,16 +69,35 @@ describe('Test Handler', function() {
     });
   });
 
-  it('should generate a merged json log', function () {
-    var opts = options.default;
+  it('should generate a merged json log by default', function () {
+    var opts = defaultOptions;
     this.timeout(timeoutMs);
-    fs.ensureDir(opts.logDir);
-    let mergedFilePath = path.join(opts.logDir, 'merged', 'results.json');
+    let mergedFilePath = defaultOptions.mergedLog;
+    // Delete the file so that previous runs of the unit test don't cause false positives
+    try {
+      fs.removeSync(mergedFilePath);
+    } catch (e) {}
 
     let cukeRunner = new TestHandler(opts);
     return cukeRunner.run().then(() => {
       chai.expect(fs.existsSync(mergedFilePath)).to.be.true;
     });
   });
 
+  it('should not generate a merged json log if the mergedLog option is set to ""', function () {
+    var opts = defaultOptions;
+    this.timeout(timeoutMs);
+    let mergedFilePath = defaultOptions.mergedLog;
+    opts.mergedLog = "";
```

Comment:
You're mutating `defaultOptions` here. Maybe that's safe with the way the test framework works, not sure.